code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -132,9 +137,10 @@
 /obj/item/weapon/reagent_containers/food/snacks/grown/mushroom/plumphelmet
 	seed = /obj/item/seeds/plump
 	name = "plump-helmet"
-	desc = "<I>Plumus Hellmus</I>: Plump, soft and s-so inviting~"
```

Comment:
I could add a tidbit about it's ale properties, the mushroom itself is a direct DF ref though.